test/suites/cursor.ts

```diff
@@ -761,7 +766,7 @@ describe('Cursor API', function() {
     const tree = new Baobab(state);
 
     it('should be possible to tell whether cursor is root.', function() {
-      assert(tree.select('one').up().isRoot());
+      assert(tree.select('one').up()!.isRoot());
```

Comment:
What does this mean? Is this some TS syntax?

Reply:
I discovered it yesterday : It is the **non-null type guard** ([source](https://www.typescriptlang.org/docs/handbook/release-notes/typescript-2-0.html#non-null-and-non-undefined-type-guards)), and allow me to tell TypeScript that I know that `tree.select('one').isUp()` will return something and not `null` (as it can, according to its signature).